iwyu.cc

```diff
@@ -3997,6 +3999,22 @@ int main(int argc, char **argv) {
   std::unique_ptr<clang::CompilerInstance> compiler(CreateCompilerInstance(
       options_parser.clang_argc(), options_parser.clang_argv()));
   if (compiler) {
+    // Figure out the absolute path to the resource-dir, which might be relative
+    // to the install directory
+    StringRef Executable = argv[0];
+    llvm::SmallString<128> AbsoluteResourceDir(IWYU_RESOURCE_DIR);
+    llvm::SmallString<128> P = llvm::sys::path::parent_path(Executable);
+    llvm::sys::path::append(P, "..");
+    if (std::error_code EC =
+            llvm::sys::fs::make_absolute(P, AbsoluteResourceDir))
```

Comment:
What happens here if `AbsoluteResourceDir` is in fact absolute?

Reply:
Nothing.